specification/src/main/asciidoc/platform/BrandMapping.adoc

```diff
@@ -0,0 +1,114 @@
+[cols=2, options=header]
+.Java EE to Jakarta EE Mapping
+|===
+|Java EE Technology
+|Jakarta EE Technology
+
+|Java™ Platform, Enterprise Edition 8
+|Jakarta™ EE Platform 8
+
+|Enterprise JavaBeans™ 3.2
+|Jakarta™ Enterprise Beans 3.2
+
+|JavaServer Pages™ 2.3
+|Jakarta™ Server Pages 2.3
+
+|Expression Language 3.0
+|Jakarta™ Expression Language 3.0
+
+|Common Annotations for the Java Platform 1.3
+|Jakarta™ Annotations 1.3
+
+|Java™ Servlet 4.0
+|Jakarta™ Servlet 4.0
+
+|Java™ Message Service 2.0
+|Jakarta™ Messaging 2.0
+
+|Java™ Transaction API 1.2
+|Jakarta™ Transaction 1.3
+
+|JavaMail™ API 1.6
+|Jakarta™ Mail 1.6
+
+|JavaBeans™ Activation Framework 1.1
+|Jakarta™ Activation 1.2
+
+|Java EE™ Connector Architecture 1.7
+|Jakarta™ Connectors 1.7
+
+|Web Services for Java EE 1.4
+|Jakarta™ XML Web Services 2.3
+
+|Java™ API for XML-based RPC 1.1
+|Jakarta™ XML RPC 1.1
+
+|SOAP with Attachments API for Java™ 1.3
+|Jakarta™ SOAP with Attachments 1.4
+
+|Java™ API for XML Registries 1.0
+|Jakarta™ XML Registries 1.0
+
+|Java™ API for RESTful Web Services 2.1
+|Jakarta™ RESTful Web Services 2.1
+
+|Java™ Platform, Enterprise Edition Management 1.1
+|Jakarta™ Management 1.1
+
+|Java™ Platform, Enterprise Edition Deployment 1.2
+|Jakarta™ Deployment 1.7
+
+|Java™ Authorization Service Provider Contract for Containers 1.5
+|Jakarta™ Authorization 1.5
+
+|Java™ Authentication Service Provider Interface for Containers 1.1
+|Jakarta™ Authentication 1.1
+
+|Java™ EE Security API 1.0
+|Jakarta™ Security 1.0
+
+|Debugging Support for Other Languages 1.0
+|Jakarta™ Debugging Support for Other Languages 1.0
+
+|Standard Tag Library for JavaServer Pages 1.2
+|Jakarta™ Standard Tag Library 1.2
+
+|Web Services Metadata for the Java Platform 2.1
+|Jakarta™ Web Services Metadata 1.1
```

Comment:
Should be 2.1.

Reply:
This is not what we agreed to via David's note conversation "Other version mismatches beyond JACC".  The Java EE 8 spec references 2.1, but the maven repo indicates 1.1...  We had decided to go with the 1.1 version...

Reply:
Right, we agreed in that thread to leave it 1.1.  Our draft PR also has 1.1.  I've documented the discussion in a section dedicated to this version change.